syntax = "proto3";

package api.v1;

service AuthLoginService {
    rpc Exec(LoginRequest) returns (LoginResponse);
}

message LoginRequest {
    string username = 1;
    string password = 2;
}

message LoginResponse {
    string token = 1;
    User user = 2;
}

message User {
    string key = 1;
    string username = 2;
}
